src/AvenueClothing/Controllers/ProductController.cs

```diff
@@ -127,37 +124,5 @@ into g
 
             return productProperties;
         }
-
-        private string GetVariantFromPostData(string sku, string prefix)
-        {
-            var request = System.Web.HttpContext.Current.Request;
-            var keys = request.Form.AllKeys.Where(
-                k => k.StartsWith(prefix, StringComparison.InvariantCultureIgnoreCase));
-            var properties = keys.Select(k => new { Key = k.Replace(prefix, string.Empty), Value = Request.Form[k] })
-                .ToList();
-
-            Product product = SiteContext.Current.CatalogContext.CurrentProduct;
-            string variantSku = null;
-
-            // TODO:
-            // // If there are variant values we'll need to find the selected variant
-            // if (!product.IsVariant && properties.Any())
-            // {
-            //     var variant = product.Variants.FirstOrDefault(v => v.ProductProperties
-            //           .Where(pp => pp.ProductDefinitionField.DisplayOnSite
-            //               && pp.ProductDefinitionField.IsVariantProperty
-            //               && !pp.ProductDefinitionField.Deleted)
-            //           .All(p => properties.Any(kv => kv.Key.Equals(p.ProductDefinitionField.Name, StringComparison.InvariantCultureIgnoreCase) && kv.Value.Equals(p.Value, StringComparison.InvariantCultureIgnoreCase))));
-            //     variantSku = variant.VariantSku;
-            // }
-            //
-            // // Only use the current product where there are no variants
-            // else if (!product.Variants.Any())
-            // {
-            //     variantSku = product.Sku;
-            // }
-
-            return variantSku;
```

Comment:
Where did this section go? Is the old functionality preserved elsewhere?

Reply:
It's actually only used here these days: https://github.com/Ucommercenet/Avenue-Clothing-For-Umbraco/blob/c8c95aed15757ac097c8d2fe3dbdb033c51864ff/src/AvenueClothing/Api/AvenueClothingApiProductController.cs#L60
I think this is old remains of a former civilization.

Reply:
Used here only:

https://github.com/Ucommercenet/Avenue-Clothing-For-Umbraco/blob/c8c95aed15757ac097c8d2fe3dbdb033c51864ff/src/AvenueClothing/Api/AvenueClothingApiProductController.cs#L60

Reply:
Fair enough.